verl/trainer/ppo/ray_trainer.py

```diff
@@ -204,22 +212,23 @@ def compute_advantage(data: DataProto, adv_estimator, gamma=1.0, lam=1.0, num_re
         data.batch["advantages"] = advantages
         data.batch["returns"] = returns
     elif adv_estimator == AdvantageEstimator.GRPO:
+        # TODO: test on more adv estimator type
```

Comment:
Same here, avoid using multi_turn. Using loss_mask directly

Reply:
Support loss mask in all algorithms